Content.Shared/Actions/Components/InstantActionComponent.cs

```diff
@@ -15,6 +15,12 @@ public sealed partial class InstantActionComponent : Component
     /// <summary>
     ///     The local-event to raise when this action is performed.
     /// </summary>
-    [DataField(required: true), NonSerialized]
+    [DataField, NonSerialized]
     public InstantActionEvent? Event;
```

Comment:
Mark the singular ones as obsolete. Have the message guide them to use the new list instead.